libcontainer/configs/config_linux.go

```diff
@@ -36,10 +36,10 @@ func (c Config) HostRootUID() (int, error) {
 // different when user namespaces are enabled.
 func (c Config) HostGID(containerId int) (int, error) {
 	if c.Namespaces.Contains(NEWUSER) {
-		if c.GidMappings == nil {
+		if c.GIDMappings == nil {
```

Comment:
Do we want the same here, or are they deliberately different?

```suggestion
		if len(c.GIDMappings) == 0 {
```